api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirPersonServiceImpl.java

```diff
@@ -26,21 +27,21 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+@Getter(AccessLevel.PROTECTED)
 @Component
 @Transactional
 @Setter(AccessLevel.PACKAGE)
-public class FhirPersonServiceImpl implements FhirPersonService {
+public class FhirPersonServiceImpl extends BaseFhirService<Person, org.openmrs.Person> implements FhirPersonService {
 	
 	@Autowired
 	private FhirPersonDao fhirPersonDao;
 	
 	@Autowired
 	private PersonTranslator personTranslator;
 	
-	@Override
 	@Transactional(readOnly = true)
-	public Person getPersonByUuid(String uuid) {
-		return personTranslator.toFhirResource(fhirPersonDao.getPersonByUuid(uuid));
+	public Person get(String uuid) {
+		return personTranslator.toFhirResource(fhirPersonDao.get(uuid));
 	}
```

Comment:
You should not need to implement this here as it's taken care of in the `BaseFhirService` class.

Reply:
@ibacher  what could be missing in this code,

Reply:
The travis is complaining to make this class abstract because this has been implemented in BaseFhirService, Am thinking leaving it there would direct us to another issue

Reply:
The issue Travis is complaining about is that the `FhirPersonDao` object should be called `dao` and the `PersonTranslator` object should be called `translator` so that we "magically" override the appropriate getters.

Reply:
@i bacher am facing a challenge my git cannot push code to github, it only remains in git so to make sure am on right truck i will close this once again as the last option clone fresh and send a new PR